crates/types/src/config/worker.rs

```diff
@@ -53,13 +53,23 @@ pub struct WorkerOptions {
     pub storage: StorageOptions,
 
     pub invoker: InvokerOptions,
+
+    /// # Maximum batching size for partition processors
+    ///
+    /// The maximum number of commands a partition processor will apply in a batch. The larger this
+    /// value is, the higher the throughput and latency are.
+    max_batching_size: NonZeroUsize,
```

Comment:
perhaps we need to spell out that this is for bifrost reads somehow? `max_command_batch_size`?

Reply:
Good idea. Will update it.